modules/ROOT/pages/apikit-odatav4-extension-reference.adoc

```diff
@@ -5,70 +5,65 @@
 [[Config]]
 === Config
 
-
 Default configuration
 
-
 ==== Parameters
 
 [%header%autowidth.spread]
 |===
 | Name | Type | Description | Default Value | Required
-|Name | String | The name for this configuration. Connectors reference the configuration with this name. | | *x*{nbsp}
-| Api Definition a| String |  API definition |  | *x*{nbsp}
-| Name a| String |  The identifier of this element used to reference it in other components |  | *x*{nbsp}
+|Name | String | The name for this configuration. Connectors reference the configuration with this name. | | *x*
```

Comment:
These descriptions are similar to those in the connector docs. We have improved the wording of some of them and the improvements have been accepted by the developers. I can give you the improved descriptions but don't know if you want deal with this now,